pkg/util/externalnode/externalnode.go

```diff
@@ -17,26 +17,28 @@ package externalnode
 import (
 	"crypto/sha1" // #nosec G505: not used for security purposes
 	"encoding/hex"
+	"fmt"
 	"io"
 
 	"antrea.io/antrea/pkg/apis/crd/v1alpha1"
 )
 
 const interfaceNameLength = 5
 
-func GenExternalEntityName(externalNode *v1alpha1.ExternalNode) string {
+func GenExternalEntityName(externalNode *v1alpha1.ExternalNode) (string, error) {
 	if len(externalNode.Spec.Interfaces) == 0 {
-		return ""
+		// This should not happen since openAPIV3Schema checks it.
+		return "", fmt.Errorf("failed to get interface from ExternalNode %s", externalNode.Name)
 	}
 	// Only one network interface is supported now.
 	// Other interfaces except interfaces[0] will be ignored if there are more than one interfaces.
```

Comment:
if we don't support more than one interfaces, should we set maxItems to avoid misconfiguration?

Reply:
OK, got it. Will add maxItems.